tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+import pytest
+import toml
+
+# Constants used in tests
+PULP_FIXTURES_URL = os.environ.get("PULP_FIXTURES_URL", "https://fixtures.pulpproject.org")
+
+ENV_CONSTANTS = {
+    "PULP_FIXTURES_URL": PULP_FIXTURES_URL,
+    "FILE_REMOTE_URL": PULP_FIXTURES_URL + "/file/PULP_MANIFEST",
+    "CONTAINER_REMOTE_URL": "https://registry-1.docker.io",
+    "CONTAINER_IMAGE": "hello-world",
+    "RPM_REMOTE_URL": PULP_FIXTURES_URL + "/rpm-unsigned",
+    "ANSIBLE_COLLECTION_REMOTE_URL": "https://galaxy.ansible.com/",
+    "ANSIBLE_ROLE_REMOTE_URL": "https://galaxy.ansible.com/api/v1/roles/?namespace__name=elastic",
+}
+
+
+@pytest.fixture
+def cli_settings(tmp_path, monkeypatch):
```

Comment:
Some docstring explaining what this function is doing might be nice. It took me a while to figure it out (had to read through the pytest docs) but then again, I'm no pytest expert.

Reply:
I'll write something.

Braindump: "Create a tmp_dir; place config there, point XDG_CONFIG_HOME, supply other usefull environment vars"

Maybe, i should place the other vars into their own fixture?

Reply:
> Braindump: "Create a tmp_dir; place config there, point XDG_CONFIG_HOME, supply other usefull environment vars"

Sounds good.

> Maybe, i should place the other vars into their own fixture?

Up to you. I don't have a strong preference.

Reply:
Maybe later. I just thought the name does not really match anymore.